core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/testSets/person/DataFrameTests.kt

```diff
@@ -2383,4 +2383,26 @@ class DataFrameTests : BaseTest() {
         val newName by column<String>()
         typed.select { name into newName and age }.columnNames() shouldBe listOf("newName", "age")
     }
+
+    @Test
+    fun `api for creating GroupBy with empty groups which can be aggregated using statistics`() {
+        val df1 = dataFrameOf("a", "b")(1, "c")
+        val df2 = DataFrame.empty()
+        val groupBy = dataFrameOf(columnOf("group1", "group2") named "group", columnOf(df1, df2)).asGroupBy()
+
+        shouldThrow<IllegalStateException> {
+            groupBy.aggregate {
+                sum("a")
```

Comment:
is it illegal because..?

Reply:
Added another assertion. It's illegal because column a is not found :)